src/backups2datalad/datasetter.py

```diff
@@ -90,25 +90,47 @@ async def update_from_backup(
                         if d.embargo_status is EmbargoStatus.OPEN
                         else "private"
                     )
+        if not dandiset_ids and self.config.gh_org is not None:
```

Comment:
So we will delete only when ran on all, not specific dandisets . Should be fine for the majority of the use cases but I am afraid it might cause confusion since I will forget such peculiarity and might need/try to run on specific one(s).  Is it too convoluted to make it also work if dandiset_ids were provided?

Reply:
> Is it too convoluted to make it also work if dandiset_ids were provided?

In that case, what deletions would you expect to be detected?  Do you want all recently-deleted Dandisets to be marked private or only those that are included in the given list of IDs?  For the latter option, note that requesting a backup of a nonexistent or deleted Dandiset currently results in an error, and I currently don't see a decent way to turn that error into some "Dandiset was deleted" marker.

Reply:
I was thinking about "latter".

interesting. Ok -- so if we error upon explicit specification of a deleted dataset, then I guess it is ok as now. Thank you!